includes/pages/admin_shifts.php

```diff
@@ -422,7 +422,8 @@ function admin_shifts()
             . form_spinner(
                 'angeltype_count_' . $type->id,
                 $type->name,
-                $needed_angel_types[$type->id]
+                $needed_angel_types[$type->id],
+                ['data-radio-name="angelmode"', 'data-radio-value="manually"']
```

Comment:
I would prefer to have an object here instead of the list, similar to how the "opt" parameter is used in all the macros in form.twig.
So, it would look like this:
{'data-radio-name': 'angelmode', 'data-radio-value': 'manually'}

Reply:
changed

Reply:
Eventually (as this are only data attributes) explicitly save define them as that? So that its more `['radio-name' => 'angelmode']`

Reply:
good idea. changed